README.md

```diff
@@ -202,27 +216,33 @@ async def scrape(sdk: SDK, url: str, *args: Any, **kwargs: Any) -> None:
 
 
 if __name__ == "__main__":
-    asyncio.run(SDK.run(scrape, "https://apprhs.org/our-locations/", setup=setup))
+    asyncio.run(SDK.run(scrape, "https://apprhs.org/our-locations/", setup=setup , schema= {}))
 ```
 
-
 ## Running a Scraper
+
 You can use poetry to run a scraper. The `run` command takes the
 scraper function and the url to scrape. The `run_from_file` command
 takes the scraper function and the path to the file containing the
 urls to scrape.
 
 ```shell
-poetry run python poetry run python scrapers/medical/apprhs.py 
+poetry run python scrapers/medical/apprhs.py
 ```
 
+## Uploadint a Scraper to the platform
 
-## Submitting a PR
```

Comment:
This should be contributing to harambe,  WES should not need to run any format commands. The platform should handle that.